kube-app-testing.sh

```diff
@@ -225,6 +232,10 @@ create_kind_cluster () {
   info "Cluster created, waiting for basic services to come up"
   kubectl -n kube-system rollout status deployment coredns
 
+  # write cluster details to file to run a manual cleanup later if required.
+  echo "export CLUSTER_NAME=${CLUSTER_NAME}" > ${ENV_DETAILS_FILE}
+  echo "export CLUSTER_TYPE=${CLUSTER_TYPE}" >> ${ENV_DETAILS_FILE}
```

Comment:
cheating here a little to avoid having to provide args to the script at the force cleanup stage.